README.md

```diff
@@ -2,21 +2,24 @@
   <img src="ICON.png" width="250"/>
   <h1>teloxide</h1>
   
-  <a href="https://docs.rs/teloxide/">
-    <img src="https://img.shields.io/badge/docs.rs-v0.3.0-blue.svg">
-  </a>
   <a href="https://github.com/teloxide/teloxide/actions">
     <img src="https://github.com/teloxide/teloxide/workflows/Continuous%20integration/badge.svg">
   </a>
+  <a href="https://teloxide.netlify.com">
+    <img src="https://img.shields.io/badge/docs-master-blue)">
+  </a>
+  <a href="https://docs.rs/teloxide/">
+    <img src="https://img.shields.io/badge/docs.rs-v0.3.0-blue.svg">
+  </a>
```

Comment:
What's the logical difference between these two badges?

Reply:
First is for docs built from `master` branch, whereas the later one - from latest release.

Reply:
Okay, nice.